packages/patternfly-4/react-core/src/components/Accordion/AccordionToggle.tsx

```diff
@@ -23,16 +25,20 @@ export const AccordionToggle: React.FunctionComponent<AccordionToggleProps> = ({
   ...props
 }: AccordionToggleProps) => (
   <dt>
-    <h3>
-      <button
-        id={id}
-        className={css(styles.accordionToggle, isExpanded && styles.modifiers.expanded, className)}
-        {...props}
-        aria-expanded={isExpanded}
-      >
-        <span className={css(styles.accordionToggleText)}>{children}</span>
-        <AngleRightIcon className={css(styles.accordionToggleIcon)} />
-      </button>
-    </h3>
+    <AccordionContext.Consumer>
+      {(TitleLevel: any) => (
```

Comment:
Sorry to be confusing here, but I think this is actually better off staying as `HeadingLevel` like it was before, since that's the name of the prop that is passed into Accordion, and it'll be more clear where this came from. `TitleLevel` is just the object a consumer can use to get predefined values for what they should pass into the `headingLevel` prop. I don't think you need any reference to `TitleLevel` at all in these components, you could use it maybe just in a unit test (e.g.`<Accordion headingLevel={TitleLevel.h2}> ...`)

Reply:
But even for the unit test, since you'd have to import it from over in the Title folder, I don't think you necessarily need to use `TitleLevel` in this PR at all.